src/utils/__tests__/MultibaseEncoder.test.ts

```diff
@@ -0,0 +1,36 @@
+import { MultibaseEncoder } from '../MultibaseEncoder'
+import { Buffer } from 'buffer'
+import { BufferEncoder } from '../BufferEncoder'
+
+const validData = Buffer.from('Hello World!')
+const validMultibase = 'zKWfinQuRQ3ekD1danFHqvKRg9koFp8vpokUeREEgjSyHwweeKDFaxVHi'
+const invalidMultibase = 'gKWfinQuRQ3ekD1danFHqvKRg9koFp8vpokUeREEgjSyHwweeKDFaxVHi'
+
+describe('multibase', () => {
```

Comment:
We often try to match the describes with the class and method names:

```suggestion
describe('MultiBaseEncoder', () => {
	describe('encode()', () => {
	})
})
```